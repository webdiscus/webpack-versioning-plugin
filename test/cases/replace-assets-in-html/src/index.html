<html>
    <head>
        <link type="text/css" href="/assets/style.css">
        <script src="/assets/script.js"></script>
    </head>
    <body>
        Hello world!
    </body>
</html>