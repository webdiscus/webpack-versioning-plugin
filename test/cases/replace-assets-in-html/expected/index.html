<html><head><link href="/assets/style.777312cffc01c1457868.css"><script src="/assets/script.666f2b8847021ccc7608.js"></script></head><body>Hello world!</body></html>